packages/expo-cli/src/commands/build/ios/credentials/prompt/index.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash';
+
+import getFromParams from './paramsProvided';
+import promptForCredentials from './userProvided';
+import promptForOverrides from './expoManaged';
+import * as consts from '../constants';
+import log from '../../../../../log';
+import _prompt from '../../../../../prompt';
```

Comment:
This import makes me think the `prompt` function defined `commands/build/ios/credentials/prompt/index.js` should perhaps have a more descriptive name like `promptForCredentials`, to avoid this naming conflict and to better describe what it does.

The `../../../../../` also makes me think maybe this hierarchy could be flattened a bit?

Reply:
1. I think that `prompt` name is descriptive enough. It's inside of the `credentials` directory.
2. How would you flatten this import without introducing absolute imports in the whole project? I believe it's not the right place for such a feature.

Reply:
I didn’t mean to introduce absolute imports, but maybe use something like `commands/build/ios/promptForCredentials.js` as the filename. However there’s no correct answer here and I see our opinions differ in this case. You can make the final decision on this, since I don’t feel that strongly about it.